pkg/landscaper/installations/executions/template/gotemplate.go

```diff
@@ -480,3 +483,23 @@ func resolveComponents(defaultCD *cdv2.ComponentDescriptor, list *cdv2.Component
 
 	return components, nil
 }
+
+func getImageVectorGoFunc(cd *cdv2.ComponentDescriptor, list *cdv2.ComponentDescriptorList) func(args ...interface{}) string {
+	return func(args ...interface{}) string {
```

Comment:
think we should return the go struct. so `map[string]interface{}`.
If someone wants the string then he could just write a yaml string or another to Yaml function.

Reply:
We added some documentation and changed the interface.